lstchain/tools/lstchain_create_irf_files.py

```diff
@@ -181,13 +218,19 @@ def setup(self):
 
         filename = self.output_irf_file.name
         if not (filename.endswith('.fits') or filename.endswith('.fits.gz')):
-            raise ValueError("f{filename} is not a correct compressed FITS file name (use .fits or .fits.gz).")
+            raise ValueError(
+                f"{filename} is not a correct compressed FITS file name"
+                "(use .fits or .fits.gz)."
+                )
 
         if self.input_proton_dl2 and self.input_electron_dl2 is not None:
             self.only_gamma_irf = False
         else:
             self.only_gamma_irf = True
 
+        if self.optimize_gh or self.optimize_th:
```

Comment:
No! One can use the one without the other. It's perfectly valid to apply energy dependent gammaness cut but use a fixed theta squared cut and vice versa!

Reply:
If these are needed to be the same, why use two options? For that case, just use a single `--energy-dependent` flag

Reply:
> No! One can use the one without the other. It's perfectly valid to apply energy dependent gammaness cut but use a fixed theta squared cut and vice versa!

Oh! I didn't realise that that is the case. So, I will just rename these 2 flags, and have them be independent of each other's value.

Would `energy-dependent-gh` and `energy-dependent-theta` be a better substitute for the names?